### PR TITLE
Plone base overhaul

### PR DESCRIPTION
**Pull request by jensens**
- isort, black, pyupgrade
- remove six
- use plone.base
- minor cleanup

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: setup.py [hunk 1]
```diff
@@ -38,25 +38,8 @@
...
-        "zope.lifecycleevent",
-        "zope.schema",
-        "zope.site",
-        "zope.testing",
-        "Acquisition",
-        "DateTime",
-        "Products.CMFCore>=2.4.0",
-        "Products.DCWorkflow",
-        "Products.GenericSetup >= 2.0",
+        "plone.base" "plone.memoize",
```
Comment (mauritsvanrees) [inline setup.py:41]: Missing a comma here between `plone.base` and `plone.memoize`.

**Review by mauritsvanrees — CHANGES_REQUESTED** [review]
LGTM, just one missing comma in `setup.py`.

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Review by mauritsvanrees — APPROVED** [review]

**Pull request merged**